.github/workflows/main.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     name: setup
     runs-on: ubuntu-latest
     outputs:
-      sdk-ref: ${{ inputs.sdk-ref || '0.2.10' }}
+      sdk-ref: ${{ inputs.sdk-ref || "e914fbe110a551b6011805269f5c557de84fec13" }}
```

Comment:
You could probably rebase it on the #126 branch, so you can use the 0.2.14 tag here.

(Although the Swift CI problem is still under investigation.)